### PR TITLE
Adds a pun, and fixes a tiny formatting error

### PR DESCRIPTION
**Pull request by tjacobdesign**
A little more dated of a pun than when I first heard it, but I thought it was worth a pull request, anyway. Hopefully it'll at least give you a chuckle as well as a cringe 😏

### CODE REVIEW

**Comment by wesbos** [comment]
Change it to significant other and we have a deal :)

**Comment by tjacobdesign** [comment]
Fair enough! :)

File: readme.md [hunk 1]
```diff
@@ -220,4 +220,6 @@ As in: OOPs I used Java!
 
 ---
 
+**Q:** How does a programmer open a jar for his significant other?
 
+**A:** He installs Java
```
Comment (dubrie) [inline readme.md]: suggest changing to "they install Java"

File: readme.md [hunk 2]
```diff
@@ -249,3 +249,9 @@ As in: OOPs I used Java!
 **A:** man man
 
 ---
+
+**Q:** How does a programmer open a jar for his significant other?
```
Comment (styfle) [inline readme.md]:
```suggestion
**Q:** How does a programmer open a jar for their significant other?
```

**Pull request merged**